client/my-sites/theme/main.jsx

```diff
@@ -94,23 +114,36 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
-	renderSectionNav( section ) {
+	renderSectionNav( currentSection ) {
 		const filterStrings = {
-			details: i18n.translate( 'Details', { context: 'Filter label for theme content' } ),
-			documentation: i18n.translate( 'Documentation', { context: 'Filter label for theme content' } ),
+			overview: i18n.translate( 'Overview', { context: 'Filter label for theme content' } ),
+			setup: i18n.translate( 'Setup', { context: 'Filter label for theme content' } ),
 			support: i18n.translate( 'Support', { context: 'Filter label for theme content' } ),
 		};
 
+		const { selectedSite, id, supportDocumentation } = this.props;
+		const siteSlug = selectedSite ? selectedSite.slug : '';
```

Comment:
Let's not introduce new references to extended `site` object properties -- please use the `getSiteSlug` selector instead :-)